<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFrontend2</class>
 <widget class="QMainWindow" name="CFrontend2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QListWidget" name="listToConvert">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="btnTogleRememberLastDfms">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string comment="Remember List" extracomment="Remember List">Remember Files</string>
          </property>
          <property name="statusTip">
           <string comment="Remember List" extracomment="Remember List">Toggle to remember Files.</string>
          </property>
          <property name="text">
           <string>R</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnAddFile">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnRemoveFile">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>21</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>9</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnConvert">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>8</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>Convert</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPreview">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Preview</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClose">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="CLogger" name="logger" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>992</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_File"/>
    <addaction name="actionRemove_File"/>
    <addaction name="separator"/>
    <addaction name="actionRemember_Last_Directory"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_File"/>
   <addaction name="actionRemove_File"/>
   <addaction name="separator"/>
   <addaction name="actionRemember_Last_Directory"/>
  </widget>
  <action name="actionAdd_File">
   <property name="text">
    <string>Add File</string>
   </property>
  </action>
  <action name="actionRemove_File">
   <property name="text">
    <string>Remove File</string>
   </property>
  </action>
  <action name="actionRemember_Last_Directory">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Remember Last Directory</string>
   </property>
   <property name="toolTip">
    <string>Remember Last Dfms</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CLogger</class>
   <extends>QWidget</extends>
   <header>clogger.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>CFrontend2</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>950</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnTogleRememberLastDfms</sender>
   <signal>clicked()</signal>
   <receiver>CFrontend2</receiver>
   <slot>togleRememberLastDfms()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>828</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>991</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemember_Last_Directory</sender>
   <signal>toggled(bool)</signal>
   <receiver>CFrontend2</receiver>
   <slot>togleRememberLastDfms()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAdd_File</sender>
   <signal>triggered()</signal>
   <receiver>btnAddFile</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>818</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove_File</sender>
   <signal>triggered()</signal>
   <receiver>btnRemoveFile</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>818</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>togleRememberLastDfms()</slot>
 </slots>
</ui>
